config/data_sources/consultation/01_interactions.yml

```diff
@@ -0,0 +1,5 @@
+load_sequence:         1
```

Comment:
This file is generated by a developer and can be modeled after other existing data sources of the same type such as CSV or external database.